tests/sample_project/.therapist.yml

```diff
@@ -6,3 +6,5 @@ actions:
       - "*.py"
     exclude: ".ignore*"
     run: ./scripts/lint.py {files}
+plugins:
+  simple: ~
```

Comment:
This seems confusing to me. I think it would be better to do `null` instead of `~`. I had to go look up `~`. Alternatively, maybe requiring an empty dict with `{}` would be the least confusing? That would simplify the code too.